### ISSUE TITLE
Monster LED Strip not controlled

### ISSUE THREAD

**Issue by scoobaspeaz** [AMoo-Miki/homebridge-tuya-lan#206]
I am having an issue where I cannot for the life of me get home bridge to do anything with this LED strip. I have its key and its ID configured. Homebridge status says its connected but if I click off or on it doesnt do anything. It has no idea what the LED strips status is.

[11/23/2020, 4:13:10 AM] [TuyaLan] Discovered Kenzies LED Strip (eb03d0c3f622c37495by78)
[TuyaAccessory] Discovery ended.
[11/23/2020, 4:13:10 AM] [TuyaLan] Connected to Kenzies LED Strip

What I picked for type is below:
"type": "RGBTWLight",

Any additional info I can give I will be glad to.

**Comment by scoobaspeaz** [comment]
Also if I do anything with my Tuya Smart app it will show things like this in the console of HomeBridge:

[TuyaAccessory] Odd message from Kenzies LED Strip with command 8: { cmd: 8, data: 'SZÌÙv9Õ~\x98<ÆéÁÆÂ·' }
[TuyaAccessory] Odd message from Kenzies LED Strip with command 8: { cmd: 8, data: 'Q{[_ã`øz\x19P\x10vD\x00¼£' }
[TuyaAccessory] Odd message from Kenzies LED Strip with command 8: { cmd: 8, data: '¿¡z\x837C\x96¢r7\x96»\\n\x8CO' }

It seems like its connected and wanting to work but something is just off.

**Comment by scoobaspeaz** [comment]
I do know home bridge sees it like a light switch instead of something like a bulb. I only have options to turn it on and off but that doesnt work either.

**Comment by scoobaspeaz** [comment]
Here is the Schema for it


Code | Type | Values
-- | -- | --
switch_led | Boolean | {true,false}
work_mode | Enum | {"range":["white","colour","scene","music"]}
bright_value_v2 | Integer | {"min":10,"scale":0,"unit":"","max":1000,"step":1}
colour_data_v2 | Json | {}
scene_data_v2 | Json | {}
countdown_1 | Integer | {"unit":"","min":0,"max":86400,"scale":0,"step":1}
control_data | Json | {}

**Comment by ivicii** [comment]
I have the same schema now for a bulb. It also does not work. So hopefully there is a solution

**Comment by ivicii** [comment]
I got it work by adding the bulbs to the smart life app and then via the homebridge-tuya-web api. 
Works perfect now:)

hmm after some testing. It works but it does not work perfectly. sometimes resets colors etc.
So i am going to try some other things

**Comment by yleechy** [comment]
Wait does ur led strip work?
and if it does work can u controll the color? 
also if that's true can u share your config file with me? cause I always mess em up it would be cool to copy n paste
thanks!

**Comment by scoobaspeaz** [comment]
> Wait does ur led strip work?
> 
> and if it does work can u controll the color? 
> 
> also if that's true can u share your config file with me? cause I always mess em up it would be cool to copy n paste
> 
> thanks!

I think he is talking about the web version and not the lan version. I have it working with the web version too. This plugin, the lan version, mine doesn't work with.

**Comment by yleechy** [comment]
What’s the difference? Do they both support HomeKit?

**Comment by ivicii** [comment]
yes it is another plugin for homebridge. But for me the LAN plugin feels more stable.
Sometimes when i set a scene in homekit with the web plugin the lights are not changing correctly. With the LAN pluging I never had those problems. So hopefully it will work for LAN soon

**Comment by scoobaspeaz** [comment]
I'm wanting the lan version to work so I'm not reliant on a stable internet connection for it to work. The web version, while works great for me, requires internet access at all times or the device won't work. Lan, this plugin, does not :)
